src/ILCompiler.Compiler/src/Compiler/JitHelper.cs

```diff
@@ -207,6 +215,20 @@ static public void GetEntryPoint(TypeSystemContext context, JitHelperId id, out
                     mangledName = "RhpFltRem";
                     break;
 
+                case JitHelperId.PInvokeBegin:
+                    mangledName = "RhpPInvoke";
+                    break;
+                case JitHelperId.PInvokeEnd:
+                    mangledName = "RhpPInvokeReturn";
+                    break;
+
+                case JitHelperId.ReversePInvokeEnter:
+                    mangledName = "RhpReversePInvoke2";
```

Comment:
It is the name of the right helper in the runtime impl (https://github.com/dotnet/corert/blob/aca349f536432b7dec2ef2e4dc6dc7c9ed42848e/src/Native/Runtime/amd64/PInvoke.asm#L106). 

The helper without "2" has a special calling convention to allow the code generator to produce a bit more efficient code, but teaching RyuJIT how to take advantage of it is more involved.